addons/mail/static/src/components/chatter_container/chatter_container.js

```diff
@@ -25,25 +25,23 @@ class ChatterContainer extends Component {
     constructor(...args) {
         super(...args);
         this.chatter = undefined;
-        this._wasMessagingInitialized = false;
         useStore(props => {
-            const isMessagingInitialized = this.env.isMessagingInitialized();
-            if (!this._wasMessagingInitialized && isMessagingInitialized) {
```

Comment:
I'm going to reintroduce this condition because it current code chatter will get updated (without change) at every single store change, which is not good.